---
content_type: page
title: Lecture Notes
uid: d962606f-c6bc-392d-903c-88aa7e027045
---

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
SES #
{{< thclose >}}
{{< thopen >}}
TOPICS
{{< thclose >}}
{{< thopen >}}
LECTURE SUMMARIES
{{< thclose >}}
{{< thopen >}}
LECTURE NOTES
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
1
{{< tdclose >}}
{{< tdopen >}}
Introduction
{{< tdclose >}}
{{< tdopen >}}
We will get to know each other and spend some time talking about how to read journal articles. We will broadly discuss the process of directed evolution for enzyme engineering and what basic steps are involved in this approach.
{{< tdclose >}}
{{< tdopen >}}


Notes ([PDF]({{< baseurl >}}/resources/ses1_ln))

Slides ([PDF]({{< baseurl >}}/resources/ses1_slides))


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
2
{{< tdclose >}}
{{< tdopen >}}
Library generation by point mutation
{{< tdclose >}}
{{< tdopen >}}
We will begin our discussion of enzyme evolution with the topic of how to introduce mutations into the starting genes. One class of techniques is those relying on point mutations. We will discuss the error-prone polymerase chain reaction and oligonucleotide-directed mutagenesis. In principle, these methods lead to the production of an enzyme library that contains members having any of the 20 different amino acids at one specific position within a given protein.
{{< tdclose >}}
{{< tdopen >}}


Notes ([PDF]({{< baseurl >}}/resources/ses2_ln))

Slides ([PDF]({{< baseurl >}}/resources/ses2_slides))


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
3
{{< tdclose >}}
{{< tdopen >}}
Library generation by recombination
{{< tdclose >}}
{{< tdopen >}}
We will continue our discussion of library generation examining methods that are based on recombination, meaning DNA is broken and rejoined in new combinations. DNA shuffling, the best known method of recombination, allows useful mutations from multiple genes to be combined. Many methods have been developed based on this principle.
{{< tdclose >}}
{{< tdopen >}}


Notes ([PDF]({{< baseurl >}}/resources/ses3_ln))

Slides ([PDF]({{< baseurl >}}/resources/ses3_slides))


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
4
{{< tdclose >}}
{{< tdopen >}}
Alternative methods for library generation
{{< tdclose >}}
{{< tdopen >}}
There are a few methods for making a library that don't fit into either category previously discussed. We will read about two of these methods: chemical mutagenesis and the use of mutator strains. We will discuss why these methods are not as general or as widely used as those discussed previously.
{{< tdclose >}}
{{< tdopen >}}


Notes ([PDF]({{< baseurl >}}/resources/ses4_ln))

Slides ([PDF]({{< baseurl >}}/resources/ses4_slides))


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
5
{{< tdclose >}}
{{< tdopen >}}
Enzyme evolution by genetic complementation
{{< tdclose >}}
{{< tdopen >}}
In enzyme evolution, it is important to link the gene of interest, the enzyme it encodes and the product of the activity of the enzyme. Many methods exist for the selection of particular library members based on enzyme activity. We will begin be examining _in vivo_ selection systems, which link cell survival to enzyme activity. Genetic complementation, the most basic of these selection systems, is the rescuing of a specific genetic defect leading to a particular phenotype by the addition of DNA encoding a gene of interest. Genetic complementation is often used to investigate the function of genes for which no function has previously been described.
{{< tdclose >}}
{{< tdopen >}}


Notes ([PDF]({{< baseurl >}}/resources/ses5_ln))

Slides ([PDF]({{< baseurl >}}/resources/ses5_slides))


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
6
{{< tdclose >}}
{{< tdopen >}}
Enzyme evolution by chemical complementation
{{< tdclose >}}
{{< tdopen >}}
Genetic complementation is often limited by the natural reactions one must use for selection. Chemical complementation is an extension of genetic complementation where a small molecule is used to reverse a genetically induced phenotypic change in intact cells. A chemical complementation strategy was developed using a small molecule to link a gene of interest to readout of enzyme activity via transcription of a reporter gene in a reaction-independent manner.
{{< tdclose >}}
{{< tdopen >}}


Notes ([PDF]({{< baseurl >}}/resources/ses6_ln))

Slides ([PDF]({{< baseurl >}}/resources/ses6_slides))


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
7
{{< tdclose >}}
{{< tdopen >}}
Enzyme evolution using phage display
{{< tdclose >}}
{{< tdopen >}}
Phage display technology was originally designed and used for the selection of proteins with high affinity for a particular ligand. Specifically, a library of proteins can be displayed on the surface of filamentous phage by fusion to a phage coat protein; fusion proteins are thereby accessible to ligands present in the external solution. Phage capable of binding a specific ligand may be retrieved using affinity purification and then used to infect _E. coli_ for expression and characterization of the encoded library member. In recent years, phage display has become a useful tool for the evolution of catalytic function. We will focus first on understanding the basics of this technique and then examine a case using indirect selection based on affinity of a suicide substrate – a substrate that is irreversibly bound in an enzyme's active site.
{{< tdclose >}}
{{< tdopen >}}
 
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
8
{{< tdclose >}}
{{< tdopen >}}
Enzyme evolution using phage display (cont.)
{{< tdclose >}}
{{< tdopen >}}
Affinity selection for improved catalysts has limitations; not all enzymatic activities have suitable suicide substrates. Direct selection of enzymes on the basis of catalysis is possible using phage display technology, but is more complex as a physical link between the phage-bound enzyme and the substrate must be established. We will discuss how this type of selection is achieved using two different examples of substrate linkage
{{< tdclose >}}
{{< tdopen >}}


Notes ([PDF]({{< baseurl >}}/resources/ses8_ln))

Slides ([PDF]({{< baseurl >}}/resources/ses8_slides))


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
9
{{< tdclose >}}
{{< tdopen >}}
Enzyme evolution using bacterial cell surface display
{{< tdclose >}}
{{< tdopen >}}
Cell surface display is a technique for presenting proteins on the surface of bacteria by fusing the proteins to surface anchoring motifs or carrier proteins. Cells are typically screened using optical methods, like fluorescence microscopy or fluorescence assisted cell sorting (FACS).
{{< tdclose >}}
{{< tdopen >}}


Notes ([PDF]({{< baseurl >}}/resources/ses9_ln))

Slides ([PDF]({{< baseurl >}}/resources/ses9_slides))


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
10
{{< tdclose >}}
{{< tdopen >}}
Enzyme evolution using yeast surface display
{{< tdclose >}}
{{< tdopen >}}
Yeast surface display is similar to bacterial cell surface display in that a protein of interest is anchored to the membrane of the cell allowing for _in vitro_ screening. A significant feature of the yeast surface display system is that it uses a eukaryotic host possessing the secretory and glycosylation machinery necessary for efficient oxidative protein folding and N-linked glycosylation. We will discuss yeast surface display and compare its utility with bacterial surface display.
{{< tdclose >}}
{{< tdopen >}}


Notes ([PDF]({{< baseurl >}}/resources/ses10_ln))

Slides ([PDF]({{< baseurl >}}/resources/ses10_slides))


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
11
{{< tdclose >}}
{{< tdopen >}}
Enzyme evolution using ribosome display
{{< tdclose >}}
{{< tdopen >}}
Ribosome display is a completely cell-free method for library selection. The physical link between genotype and phenotype in ribosome display is achieved by mRNA/ribosome/protein complexes formed in the translation step, which are directly used for selection. We will discuss the advantages of cell-free selection in the context of this technique.
{{< tdclose >}}
{{< tdopen >}}


Notes ([PDF]({{< baseurl >}}/resources/ses11_ln))

Slides ([PDF]({{< baseurl >}}/resources/ses11_slides))


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
12
{{< tdclose >}}
{{< tdopen >}}
Enzyme evolution by _in vitro_ compartmentalization
{{< tdclose >}}
{{< tdopen >}}
_In vitro_ compartmentalization (IVC) is based on water-in-oil emulsions where surfactants aid in the formation of microscopic aqueous compartments. These compartments simulate cellular compartments in which only the reaction for selection is performed. Inside each droplet transcription, translation, and expression of the desired protein takes place using active components. This method is poised to become the most promising for the future of enzyme evolution – we'll discuss why.
{{< tdclose >}}
{{< tdopen >}}


Notes ([PDF]({{< baseurl >}}/resources/ses12_ln))

Slides ([PDF]({{< baseurl >}}/resources/ses12_slides))


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
13
{{< tdclose >}}
{{< tdopen >}}
Alternative methods for enzyme/catalyst design
{{< tdclose >}}
{{< tdopen >}}
We will wrap up our discussion about directed evolution and compare and contrast the methods we have learned about in previous weeks. This week we will also spend time looking at approaches for enzyme evolution based on rational design and discuss the advantages or disadvantages of these methods as compared with directed evolution.
{{< tdclose >}}
{{< tdopen >}}


Notes ([PDF]({{< baseurl >}}/resources/ses13_ln))

Slides ([PDF]({{< baseurl >}}/resources/ses13_slides))


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
14
{{< tdclose >}}
{{< tdopen >}}
Student presentations
{{< tdclose >}}
{{< tdopen >}}
During this last class session each student will give a presentation about his or her chosen paper. We will follow these presentations with a discussion about the course and a written course evaluation.
{{< tdclose >}}
{{< tdopen >}}
 
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}